README.md

```diff
@@ -1,2 +1,51 @@
-Hello-World
-In this project, you will set up a "Hello world" repository.
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello-world
+
+> This project is my first ever on microverse and its a linter setup as well as a hello world
+
+![screenshot](./app_screenshot.png)
+
+This is a simple hello world project
+
+## Built With
+
+- HTML
+- CSS
+
+<!-- ## Live Demo
+
+[Live Demo Link](https://livedemo.com) -->
+
+
+## Getting Started
+
+
+you can clone and play around 
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/BereketRetta)
+- Twitter: [@twitterhandle](https://twitter.com/bekiopia)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/bereketretta)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+<!-- Feel free to check the [issues page](../../issues/). -->
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Last but not least, kindly add your [License](https://www.youtube.com/watch?v=gJZGLZSzP_I) 👈